templates/definition/meter/fronius-solarapi-v1.yaml

```diff
@@ -44,7 +45,7 @@ render: |
         - content-type: application/json
         auth:
           type: digest
-          user: {{ .user }}
+          user: {{- if .user }} {{ .user }}{{- else }} customer{{- end }}
```

Comment:
Rather set customer as default above

Reply:
Good point. Can I still leave the parameter as `required: false`? And, aren't there any side-effects to `usage=grid` and `usage=pv`?

Reply:
Its not used anywhere else?

Reply:
No, it is solely used in case of `usage=battery` and also only if users want to control `batterymode`.